external-crates/move/crates/move-analyzer/trace-adapter/out/adapter.js

```diff
@@ -0,0 +1,435 @@
+"use strict";
```

Comment:
Should these files be included?

Reply:
Nope... Accidental `git add .` which will now not do that anymore as I added .gitignore files to both TS projects